images/metalk8s-alert-logger/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/alertmanager/template"
+)
+
+func main() {
+	address := flag.String("address", ":19094", "address and port of service")
+	flag.Parse()
+
+	log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
+
+	http.HandleFunc("/", logAlert)
+	http.HandleFunc("/ready", serverIsRunning)
+	http.HandleFunc("/health", serverIsRunning)
+	if err := http.ListenAndServe(*address, nil); err != nil {
+		log.Fatalf("Failed to start HTTP server: %v", err)
+	}
+}
+
+func serverIsRunning(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusNoContent)
+}
+
+func logAlert(w http.ResponseWriter, r *http.Request) {
+	var alerts template.Data
+
+	if err := json.NewDecoder(r.Body).Decode(&alerts); err != nil {
+		log.Printf("Unable to parse HTTP body: %s", err)
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	for _, alert := range alerts.Alerts {
+		encoded_alert, _ := json.Marshal(alert)
+	        log.Println(string(encoded_alert))
```

Comment:
Some tab was replaced with spaces apparently:

```suggestion
		log.Println(string(encoded_alert))
```

Hints that we should add this file to `doit lint:go` :wink: